### PR TITLE
Feature/fad 5587

### PR DESCRIPTION
**Pull request by bizob2828**

### CODE REVIEW

File: src/components/support/components/SupportForm.module.scss [hunk 1]
```diff
@@ -12,7 +12,16 @@
...
 
-.CancelBtn {
+.CancelBtn, .ToggleButton {
   float: right;
 }
 
+.Results {
+  max-height: rem(400);
+  overflow-y: scroll;
+  margin-top: rem(5)gT
```
Comment (jonambas) [inline src/components/support/components/SupportForm.module.scss]: dont need any margin top, I don't think

File: src/components/support/components/SupportForm.module.scss [hunk 2]
```diff
@@ -12,7 +12,16 @@
   align-items: center;
 }
 
-.CancelBtn {
+.CancelBtn, .ToggleButton {
   float: right;
 }
 
+.Results {
+  max-height: rem(400);
```
Comment (jonambas) [inline src/components/support/components/SupportForm.module.scss]: I think you can just use `height` here, the support panel doesnt scale at all - always a fixed height

Comment (jonambas) [reply src/components/support/components/SupportForm.module.scss]: `height: rem(425);` fills the panel in nicely

File: src/components/support/components/SupportForm.module.scss [hunk 3]
```diff
@@ -12,7 +12,19 @@
...
   float: right;
 }
 
+.Results {
+  height: rem(425);
+  overflow-y: scroll;
+}
+
+.Results ul {
+  list-style-type: none;
```
Comment (jonambas) [inline src/components/support/components/SupportForm.module.scss]: Add
```css
padding: 0;
margin: 0;
```

File: src/components/support/components/AlgoliaResults.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { UnstyledLink, Icon } from '@sparkpost/matchbox';
+import { Snippet } from 'react-instantsearch/dom';
+
+const AlgoliaResults = ({ hit }) => (
+  <div>
+    <strong><Icon name='Link' /><UnstyledLink to={hit.permalink}>{hit.post_title}</UnstyledLink></strong>
```
Comment (jonambas) [inline src/components/support/components/AlgoliaResults.js]: Can we add a space here between the icon and link

File: src/components/support/components/AlgoliaResults.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { UnstyledLink, Icon } from '@sparkpost/matchbox';
+import { Snippet } from 'react-instantsearch/dom';
+
+const AlgoliaResults = ({ hit }) => (
+  <div>
```
Comment (jonambas) [inline src/components/support/components/AlgoliaResults.js]: add this class name: 
```
className={styles.Result}

@import '~@sparkpost/matchbox/src/styles/config.scss';

.Result {
  margin-bottom: spacing();
}
```

File: src/components/support/components/AlgoliaResults.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { UnstyledLink, Icon } from '@sparkpost/matchbox';
+import { Snippet } from 'react-instantsearch/dom';
+import styles from './AlgoliaResults.module.scss';
+
+const AlgoliaResults = ({ hit }) => (
+  <div className={styles.Result}>
+    <strong><Icon name='Link' /> <UnstyledLink to={hit.permalink}>{hit.post_title}</UnstyledLink></strong>
```
Comment (jasonrhodes) [inline src/components/support/components/AlgoliaResults.js]: Add `external` prop to `UnstyledLink`, and use `Component={Link}` from react-router-dom

File: src/components/support/components/SearchPanel.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+import { InstantSearch, Hits } from 'react-instantsearch/dom';
+import AlgoliaSearch from './AlgoliaSearch.js';
+import config from 'src/config';
+const searchCfg = config.support.algolia;
+
+export class SearchPanel extends Component {
+  render() {
+    const { toggleForm } = this.props;
+
+    return <InstantSearch
```
Comment (jasonrhodes) [inline src/components/support/components/SearchPanel.js]: should this wrap _everything_ here or just the 2 sections that use instantsearch components?

File: src/components/support/components/AlgoliaResults.js (hunk 6 above)
Comment (bizob2828) [reply src/components/support/components/AlgoliaResults.js]: why the component? not seeing others like this

File: src/components/support/components/SearchPanel.js (hunk 7 above)
Comment (bizob2828) [reply src/components/support/components/SearchPanel.js]: i could but I don't know why, can you explain

**Pull request merged**